packages/vuetify/src/components/VDatePicker/VDatePickerMonth.tsx

```diff
@@ -112,17 +113,7 @@ export const VDatePickerMonth = genericComponent<VDatePickerMonthSlots>()({
           rangeStop.value = adapter.endOfDay(_value)
         }
 
-        const diff = adapter.getDiff(rangeStop.value, rangeStart.value, 'days')
-        const datesInRange = [rangeStart.value]
-
-        for (let i = 1; i < diff; i++) {
-          const nextDate = adapter.addDays(rangeStart.value, i)
-          datesInRange.push(nextDate)
-        }
-
-        datesInRange.push(rangeStop.value)
-
-        model.value = datesInRange
+        model.value = [rangeStart.value, rangeStop.value]
```

Comment:
This is a breaking change.

Reply:
That's why it's to dev. Should I go next instead?